models/models.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api
+
+
+class meeting__scheduler(models.Model):
```

Comment:
@NilsNeeb Don't use two successive underscores in object names. The odoo code convention is snake case